gatsby-config.js

```diff
@@ -438,6 +438,39 @@ module.exports = {
         nodesPerFeedFile: Infinity,
       },
     },
+    {
+      resolve: `gatsby-plugin-json-output`,
+      options: {
+        siteUrl,
+        graphQLQuery: `
+        {
+          allMdx(filter: {fields: {slug: {regex: "/docs/release-notes/"}}}) {
+            nodes {
+              frontmatter {
+                releaseDate(fromNow: false)
+                version
+              }
+              fields {
+                slug
+              }
+            }
+          }
+        }
+        `,
+        serializeFeed: ({ data }) =>
+          data.allMdx.nodes
+            .map(({ frontmatter, fields }) => ({
+              agent:
+                fields.slug !== '/docs/release-notes'
+                  ? fields.slug.split('release-notes/')[1].replace('-', '')
+                  : null,
```

Comment:
This looks like it's `agent` for every result. We'll need to do a better job at identifying the agent type.

Reply:
That data should be available as the “subject” in frontmatter. Can you use that instead?

Reply:
Oh duh 🤦  Good call!